src/AddRemove.js

```diff
@@ -0,0 +1,104 @@
+class List {
+  constructor() {
+    this.list = JSON.parse(localStorage.getItem('todo-list'));
+    if (!this.list) {
+      this.list = [];
+    }
+    this.showList();
+  }
+
+  showList() {
+    this.saveData();
+    const listSection = document.getElementById('list-items');
+    listSection.innerHTML = '';
+    this.list.forEach((activity) => {
+      let activityItem = `<li class ="d-flex s-between list-item" id ="item-data-${activity.index}"> `;
+      if (activity.completed) {
+        activityItem += `<span class="material-icons done update-status" data="${activity.index}">
+              done
+            </span>
+            <p contenteditable="true" class="completed activity" data="${activity.index}">
+              ${activity.description}
+            </p>
+            `;
+      } else {
+        activityItem += `
+            <span class="material-icons  update-status"  data="${activity.index}">
+              check_box_outline_blank
+            </span>
+            <p contenteditable="true" class="activity" data="${activity.index}">
+              ${activity.description}
+            </p>`;
+      }
+      activityItem += `
+          <span class="material-icons delete-activity" data="${activity.index}">
+            delete
+          </span>
+        </li>
+      `;
+      listSection.innerHTML += activityItem;
+    });
+    this.activityActions();
+  }
+
+  saveData() {
+    for (let i = 0; i < this.list.length; i += 1) {
+      this.list[i].index = i;
+    }
+    this.list.sort((a, b) => {
+      if (a.index < b.index) {
+        return -1;
+      }
+      if (a.index > b.index) {
+        return 1;
+      }
+      return 0;
+    });
+    localStorage.setItem('todo-list', JSON.stringify(this.list));
+  }
+
+  addActivity(activity) {
+    if (activity.length > 0) {
+      const newActivity = {
+        description: activity,
+        completed: false,
+        index: this.list.length,
```

Comment:
- [ ] When a task is added its `index` property should be equal to the length of the array. So from the image below the `index` of the first item should be 1, and the index of the second item should be 2. Kindly fix this.

- [ ] After your refactor also ensure the index values are still updated upon deletion.

  ![lekan-index](https://user-images.githubusercontent.com/36542986/159693565-d4645181-d1c4-4bf2-9a47-5dd138abe625.jpg)